js/utils/trumbowyg.js

```diff
@@ -0,0 +1,29 @@
+import $ from 'jquery';
+import 'trumbowyg';
+
+$.trumbowyg.svgPath = '../node_modules/trumbowyg/dist/ui/icons.svg';
+
+export const editorOptions = {
+  btns: [
+    ['formatting'],
+    ['bold', 'italic'],
+    ['link'],
+    ['insertImage'],
+    'btnGrp-justify',
+    'btnGrp-lists',
+  ],
+};
+
+export function installRichEditor(attachPoint) {
```

Comment:
FWIW, after letting this sink in for a day, if you haven't made this change already... if you left this is as is and just changed the following line in EditListing.js, I think that would fine:

```
installRichEditor('#editListingDescription');
```

to:

```
installRichEditor(this.$('#editListingDescription'));
```